syntax = "proto3";

service HelloService {
    rpc SayHi (HelloRequest) returns (HelloResponse) {}
    rpc SayHello (HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string message = 1;
}
